pkg/eventlog/mongo/mongo.go

```diff
@@ -108,3 +115,35 @@ func (m *Log) Clear(id string) error {
 	_, err := c.RemoveAll(bson.M{"_id": bson.M{"$lte": id}})
 	return err
 }
+
+func (m *Log) newSession() (*mgo.Session, error) {
+	tlsConfig := &tls.Config{}
+
+	dialInfo, err := mgo.ParseURL(m.url)
+	if err != nil {
+		return nil, err
+	}
+
+	dialInfo.Timeout = env.MongoConnectionTimeoutSecWithDefault(1)
+
+	if m.certPath != "" {
+		roots := x509.NewCertPool()
+		cert, err := ioutil.ReadFile(m.certPath)
+		if err != nil {
+			return nil, err
+		}
+
+		if ok := roots.AppendCertsFromPEM(cert); !ok {
+			return nil, fmt.Errorf("failed to parse certificate from: %s", m.certPath)
+		}
+
+		tlsConfig.InsecureSkipVerify = true
```

Comment:
Why are we skipping verification?

Reply:
good point this should be configurable, skipping for self signed which i had during testing